apis/v1alpha1/types.go

```diff
@@ -393,6 +2201,22 @@ type TagInfoForResource struct {
 	TagList     []*Tag  `json:"tagList,omitempty"`
 }
 
+// Text transformations eliminate some of the unusual formatting that attackers
+// use in web requests in an effort to bypass detection.
+type TextTransformation struct {
+	Priority *int64  `json:"priority,omitempty"`
+	Type     *string `json:"type_,omitempty"`
```

Comment:
We can rename this json tag to `type`

Reply:
Here you go https://github.com/aws-controllers-k8s/eks-controller/blob/main/generator.yaml#L445-L446